cinema/serializers.py

```diff
@@ -62,19 +79,59 @@ class MovieSessionListSerializer(MovieSessionSerializer):
 
     class Meta:
         model = MovieSession
-        fields = (
+        fields = [
             "id",
             "show_time",
             "movie_title",
+            "tickets_available",
             "cinema_hall_name",
             "cinema_hall_capacity",
-        )
+        ]
+
+
+class TicketSerializer(serializers.ModelSerializer):
+    class Meta:
```

Comment:
it seems like you didn't want to implement an optional task about validation on the serializer level ?:(